### PR TITLE
updates needed for wm-browser to work w. dark ui

### PR DESCRIPTION
**Pull request by gvn**
Related to #553

### CODE REVIEW

File: src/pages/project/cartzoom.js [hunk 1]
```diff
@@ -42,8 +42,10 @@ module.exports = {
    * @return {int} An integer representing the scale transform
    */
   getMaxPageSize: function () {
-    var w = window.innerWidth - 60;
-    var h = window.innerHeight - 60;
+    var elWrapper = this.refs.map.getDOMNode();
```
Comment (gvn) [inline src/pages/project/cartzoom.js:45]: Using `window` was messing up positioning in browser due to the header.

File: src/pages/project/cartzoom.js [hunk 2]
```diff
@@ -120,9 +122,10 @@ module.exports = {
 
   componentDidUpdate: function () {
     var boundingBox = this.getPageBoundingRect();
-    var btnZoomOut = this.refs.btnZoomOut.getDOMNode();
```
Comment (gvn) [inline src/pages/project/cartzoom.js:123]: Browser has no zoom out button so this was breaking it...

File: src/pages/project/cartzoom.js (hunk 1 above)
Comment (k88hudson) [inline src/pages/project/cartzoom.js:45]: ah, good call!

File: src/pages/project/cartzoom.js [hunk 3]
```diff
@@ -42,8 +42,10 @@ module.exports = {
    * @return {int} An integer representing the scale transform
    */
   getMaxPageSize: function () {
-    var w = window.innerWidth - 60;
-    var h = window.innerHeight - 60;
+    var elWrapper = this.refs.map.getDOMNode();
+
+    var w = elWrapper.clientWidth - 60;
+    var h = elWrapper.clientHeight - 60;
```
Comment (k88hudson) [inline src/pages/project/cartzoom.js]: The 60 pixel offset shouldn't apply anymore now though, right?

**Comment by k88hudson** [comment]
I think the offset needs to be removed, other than that this is good to go. using a ref rather than window is a much smarter idea

**Comment by k88hudson** [comment]
R+

**Pull request merged**